python/lsst/cp/pipe/defects.py

```diff
@@ -673,7 +673,7 @@ def _markBlocksInBadColumn(self, x, y, extensionX, multipleX, defects):
                     limits.append(multipleY[gapIndex])
                     limits.append(multipleY[gapIndex+1])
                 limits.append(maxY)  # maximum last
-                assert (len(limits)%2 == 0)  # `limits` it's even by design, but check anyways
+                assert (len(limits)%2 == 0, 'limits is even by design, but check anyways')
```

Comment:
This code is not correct. I think you get a warning from Python about it always being true and flake8 also complains. You need to remove the parentheses.